example/src/App.js

```diff
@@ -72,7 +72,10 @@ function App() {
   }, [appType]);
 
   const handleAppNavOpen = (event) => {
-    setAnchorEl(event.currentTarget);
+    if (event) { 
```

Comment:
We do not have a `onClick` `event` here to open the `nav`. I would not really consider having a `uuid` in the `url` an `event`, so I added this check.